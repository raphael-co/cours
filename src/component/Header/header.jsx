import style from './header.module.css'

function Header() {

    return (



        <header className={style.hello}>
      <p className={style.Boss2}>Hello Friend</p>
      <div className={style.Boss}><p>Boss</p></div>
      </header>

    )
}
export default Header;