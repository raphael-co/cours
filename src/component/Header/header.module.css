.hello {
    overflow: hidden;
    background-color: #791cf8;
    padding: 20px 10px;
    
    }
    .Boss {
      float: right;
      color:white;
      background-color: #791cf8; 
      text-align: center;
      width: 75px;
      border-radius: 23%;
    
    }
    .Boss:hover {
      float: right;
      color:#791cf8;
      background-color: white; 
      text-align: center;
      width: 75px;
      border-radius: 23%;
    
    }
    .Boss2 {
      float: left;
      color: white;
      text-align: center;
    }