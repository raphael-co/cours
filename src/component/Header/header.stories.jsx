import Header from './header.jsx'
export default {
    title: 'component/Header',
    component : 'Header'
}

export const Normal = () => <Header></Header> 