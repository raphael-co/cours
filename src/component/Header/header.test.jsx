import { render, screen } from "@testing-library/react"
import Header from "./header.jsx"

test("print hello world to the user", () => {
    render(<Header/>);

    const head = screen.getByText("Hello Friend");
    expect(head).toBeInTheDocument();
    expect(head).toMatchSnapshot();
})