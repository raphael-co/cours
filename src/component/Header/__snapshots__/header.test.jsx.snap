// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`print hello world to the user 1`] = `
<p
  class="Boss2"
>
  Hello Friend
</p>
`;
